### PR TITLE
Update troubleshooting section

### PR DESCRIPTION
**Pull request by dz-void**

### CODE REVIEW

File: docs/troubleshooting/scripts.md [hunk 1]
```diff
@@ -27,22 +27,22 @@ Safe mode starts Windows in a very simple way, with only a few files and drivers
...
 
 ### Safe Mode with Command Prompt
 
-The Command Proompt (CMD) is available in all safe mode states, but "Safe Mode with Command Prompt" only gives you the Command Prompt. You will not have a user interface to help you get around. Instead, you will have an admin-level command prompt.
+The Command Prompt (CMD) is available in all safe mode states, but "Safe Mode with Command Prompt" only gives you the Command Prompt. You will not have a user interface to help you navigate your system. Instead, you will have an admin-level command prompt.
 
 ### Safe Mode with Networking
 
-Safe Mode with Networking functions similarly to safe mode without networking. Safe Mode with Networking adds the network drivers and services you need to connect to the internet and other computers on your network.
+Safe Mode with Networking functions similarly to safe mode without networking. B safe Mode with Networking adds the network drivers and services you need to connect to the internet.
```
Comment (PencilNavigator) [inline docs/troubleshooting/scripts.md:38]: "b safe" mistyped.

**Review by PencilNavigator — APPROVED** [review]
good, just one small change and will be good to merge

File: docs/troubleshooting/scripts.md (hunk 1 above)
Comment (dz-void) [reply docs/troubleshooting/scripts.md:38]: Sorry

**Review by PencilNavigator — APPROVED** [review]
lgtm thx void

**Pull request merged**